strapi/strapi
# feat(main-nav): Main Nav refactoring, add the Home icon

### What does it do?

In the condensed view of the Left Menu it shows the new Home icon. This new link is responsible to redirect to the home

### Why is it needed?

In the refactoring of the Main Nav we want a new home icon to have the task of connecting to the home page, while the logo must only show the image of the logo.

### How to test it?

Disclaimer: the new home Link is used only in the condensed view at the moment because I need to implement the components used in the Logo and in the new Home icon before implementing the new MainNav wrapper

Open a Strapi App
in the left Main Nav click on the reduce button, to show the condensed view of the Main Nav
you can now see the new Home icon 
if you try to click on the home icon you will be redirected to the home
Please test the themes and all the possible status of the link (on hover, active) and also the popover

You can find all the info and link to the figma file in the Jira ticket

### Related issue(s)/PR(s)

CS-764
